### ISSUE TITLE
Material - Member card styling issue

### ISSUE THREAD

**Issue by nsaneman** [Audentio/xftheme-issues#100]
If a user has selected to hide their status/activity then when viewing the member card (as a user or guest) their name will clip into their cover image.

This fixes it for me.
`.memberCard__meta {min-height: 16px;}`

**Comment by nsaneman** [comment]
So it seems as though there's more to this bug report.

If a user has a long location set then it'll also mess up the member card.

Limiting it's size might fix the problem, but it might be easier to move the time/date to the left corner and put the location under it? Just make sure you set the minimum heights in case a user isn't sharing their activity or location.

Here's what I'm using to limit the size of the location string.
```
.memberCard__meta--location div {
	text-overflow: ellipsis;
	max-width: 130px;
	overflow: hidden;
	white-space: nowrap;
}
```

**Comment by nsaneman** [comment]
This is also a problem when you've got a member that's registered with a long name.

```
.memberCard .media__body .username {
	text-overflow: ellipsis;
	max-width: 175px;
	overflow: hidden;
	white-space: nowrap;
}
```

**Comment by Ian-Hitt** [comment]
Added, thanks!

**Comment by nsaneman** [comment]
@Ian-Hitt @dprock I'm not seeing this fix in 1.5.13.1 either. Both long usernames and locations cause the whole member card to be a mess up.

**Comment by nsaneman** [comment]
@Ian-Hitt @mikecreuzer Has this been fixed for 1.5.14.0?

**Comment by Ian-Hitt** [comment]
@nsaneman We put out a soft release a few days after. You can force re-install the theme for some of these fixes, or just wait for 1.5.14.0 which should be done within the next two days.

**Comment by nsaneman** [comment]
@Ian-Hitt I'll check again. But I originally got the bad 1.5.13.1 release and manually installed it and then the next day I installed the 1.5.13.1 re-release. So you're saying that there was yet another release after that?

**Comment by nsaneman** [comment]
@Ian-Hitt I just re-downloaded and installed the theme. This fix as well as issue #107 isn't in 1.5.13.1.

**Comment by Ian-Hitt** [comment]
<img width="529" alt="screen shot 2017-06-06 at 12 35 57 pm" src="https://user-images.githubusercontent.com/15836384/26843107-b6d4c166-4ab4-11e7-9cf2-368ff9a410cc.png">

**Comment by nsaneman** [comment]
@Ian-Hitt I've just compared the XML from 1.5.13.0 to 1.5.13.1 and it contains literally zero changes for the member card name length or location length. There's also not a single change to the polls.

Change your location text to "THIS IS STILL NOT FIXED IN MATERIAL 1.5.13.1" and you'll see at least 1 of the issues.

**Comment by nsaneman** [comment]
@Ian-Hitt I don't intend to bother or frustrate you with this stuff, so I'm sorry if it's slightly annoying for you. I'm only reporting what I see here.

I promise you that if you were to install 1.5.13.1 from the UI.X theme installer you'll see the exact same issues. It's as if the public is getting the wrong version that's missing a few of the fixes, while your version seems to be fine.
